types/hyphen/hyphen-tests.ts

```diff
@@ -26,6 +27,12 @@ hyphenateEnGbAsync('hyphenation', { hyphenChar: '#' }).then(result => {
     }
 });
 
+// Test HTML sync from global English
+const hyphenateEn = hyphenateHTMLSyncEn('hyphenation', { hyphenChar: '#' });
```

Comment:
Could you remove the `hyphenateEn` variable and add the function call directly to the condition instead? I feel that `hyphenateEn` is a bit confusing and it's how it is done elsewhere in this file. Thank you.